includes/API.php

```diff
@@ -481,26 +480,6 @@ public function delete_product_item( $product_item_id ) {
 	}
 
 
-	/**
-	 * Sends Pixel events.
-	 *
-	 * @since 2.0.0
-	 *
-	 * @param string  $pixel_id pixel ID
-	 * @param Event[] $events events to send
-	 * @return Response
-	 * @throws Framework\SV_WC_API_Exception
-	 */
-	public function send_pixel_events( $pixel_id, array $events ) {
```

Comment:
This is a public method. should we perhaps deprecate it before a complete removal?

Reply:
the likelihood of this being used outside of this extension is pretty low. so I reckon this is okay.